controllers/keystoneapi_controller.go

```diff
@@ -437,12 +447,15 @@ func (r *KeystoneAPIReconciler) reconcileUpgrade(ctx context.Context, instance *
 func (r *KeystoneAPIReconciler) reconcileNormal(ctx context.Context, instance *keystonev1.KeystoneAPI, helper *helper.Helper) (ctrl.Result, error) {
 	r.Log.Info("Reconciling Service")
 
-	// If the service object doesn't have our finalizer, add it.
-	controllerutil.AddFinalizer(instance, helper.GetFinalizer())
-	// Register the finalizer immediately to avoid orphaning resources on delete
-	if err := r.Update(ctx, instance); err != nil {
-		return ctrl.Result{}, err
-	}
+	// TODO1 remove from here and use it in lib-common through recconcileIinit/ CreateOrPatchDB #line no:232
+
+	// // If the service object doesn't have our finalizer, add it.
+	// controllerutil.AddFinalizer(instance, helper.GetFinalizer())
+	// r.Log.Info("Adding Finalizer")
+	// // Register the finalizer immediately to avoid orphaning resources on delete
+	// if err := r.Update(ctx, instance); err != nil {
+	// 	return ctrl.Result{}, err
+	// }
```

Comment:
why you did this?

Reply:
a misunderstanding , so here this only removes the keystoneapi instance finalizer , and not the database one (from lib common), I though they were the same.
I'll add that back in

Reply:
ok

Reply:
we have to remove both. 1st on the db 2nd the one we had